pkg/driver/lookup.go

```diff
@@ -13,9 +12,12 @@ func Lookup(name string) (driver.Driver, error) {
 		return &DockerDriver{}, nil
 	case "debug":
 		return &driver.DebugDriver{}, nil
-	case "command":
-		return &CommandDriver{Name: name}, nil
 	default:
-		return nil, fmt.Errorf("unsupported driver: %s", name)
+		cmddriver := &CommandDriver{Name: name}
+		if cmddriver.CheckDriverExists() {
+			return cmddriver, nil
+		}
+
+		return nil, fmt.Errorf("unsupported driver or driver not found: %s", name)
```

Comment:
I recommend adding a mention that the driver is expected to be present on the PATH. Perhaps "unsupported driver or driver not found on the PATH"